app/templates/admin/sales/marketer.hbs

```diff
@@ -1 +1,40 @@
-{{outlet}}
+<div class="ui stackable grid">
+  <div class="row">
+    <h1 class="ui header six wide column">{{t 'Tickets Summary'}}</h1>
+  </div>
+  <div class="row">
+    <div class="sixteen wide column">
+      <table class="ui stackable structured celled table">
+        <thead>
+          <tr>
+            <th rowspan="2">{{t 'Marketers'}}</th>
+            <th colspan="3" class="ui green inverted segment center aligned">{{t 'Completed Orders '}}</th>
+          </tr>
+          <tr>
+            <th class="right aligned">{{t 'Tickets'}}</th>
+            <th class="right aligned">{{t 'Sales (discounted)'}}</th>
+            <th class="right aligned">{{t 'Discounted amount'}}</th>
+          </tr>
+        </thead>
+        <tbody>
+          {{#each model as |entry|}}
+            <tr>
+              <td>{{entry.marketers}}</td>
+              <td class="right aligned">{{entry.completedTickets}}</td>
+              <td class="right aligned">{{number-format entry.completedSales}}</td>
+              <td class="right aligned">{{number-format entry.completedDiscounts}}</td>
+            </tr>
+          {{/each}}
+        </tbody>
+        <tfoot>
+          <tr>
+            <th>{{t 'Total'}}</th>
+            <th class="right aligned">0</th>
```

Comment:
These values should be calculated using the data in the model. Please create an object that holds all the totals.